### PR TITLE
Reduce old testsuite time

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

File: package/yast2-devtools.spec [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
 
 Name:           yast2-devtools
-Version:        3.1.46
+Version:        3.1.47
```
Comment (lslezak) [inline package/yast2-devtools.spec]: We should not raise version to test it in YaST:Head first. And if it does not break something *then* submit to Factory.

File: package/yast2-devtools.changes [hunk 2]
```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Tue Sep 20 13:29:46 UTC 2016 - jreidinger@suse.com
+
+- run old yast testsuite paralel to speed up build (bsc#999203)
```
Comment (lslezak) [inline package/yast2-devtools.changes]: `in parallel`

Comment (mvidner) [inline package/yast2-devtools.changes]: **in** para**ll**el

File: build-tools/rpm/macros.yast [hunk 3]
```diff
@@ -79,7 +79,7 @@ fi \
     %if %{with yast_run_ci_tests} \
       if [ -f "configure.in.in" -o -f "configure.ac.in" ]; then \
           # TODO: fix the check:ci task to also work with autotools based modules \
-          %{__make} check \\\
+          %{__make} check %{?jobs:-j%jobs}%{?!jobs:%{?_smp_mflags:%_smp_mflags}} \\\
```
Comment (lslezak) [inline build-tools/rpm/macros.yast]: I can hardly understand that, but it's the same as for the normal `make`. So I'm fine with that. :wink:

Comment (mvidner) [inline build-tools/rpm/macros.yast]: `%{?SYMBOL:SUBST}` if SYMBOL is defined, use SUBST
`%{?!SYMBOL:SUBST}` if SYMBOL is **not** defined, use SUBST

Curiously, there does not seem to be an official reference for this

**Pull request merged**